rest_log/components/service.py

```diff
@@ -123,6 +123,12 @@ def _log_call_in_db_values(self, _request, *args, params=None, **kw):
             params = dict(params or {}, args=args)
 
         result = kw.get("result")
+        if isinstance(result, Response):
+            status_code = result.status_code
+            result = {"status_code": status_code}
```

Comment:
Maybe content length and filename from the header if they are available?
Also, please, add a test for this :pray: